core/src/main/java/org/apache/accumulo/core/tabletserver/log/LogEntry.java

```diff
@@ -19,67 +19,95 @@
 package org.apache.accumulo.core.tabletserver.log;
 
 import java.util.Map.Entry;
+import java.util.Objects;
 
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Value;
-import org.apache.accumulo.core.dataImpl.KeyExtent;
-import org.apache.accumulo.core.metadata.schema.MetadataSchema.TabletsSection.LogColumnFamily;
 import org.apache.hadoop.io.Text;
 
+import com.google.common.base.Preconditions;
+
 public class LogEntry {
-  private final KeyExtent extent;
-  public final long timestamp;
-  public final String filename;
 
-  public LogEntry(KeyExtent extent, long timestamp, String filename) {
-    // note the prevEndRow in the extent does not matter, and is not used by LogEntry
-    this.extent = extent;
+  private final long timestamp;
```

Comment:
Not something to do in this PR and I am not sure about it, but this timestamp may be able to be removed also.  I need to look into this.

Reply:
I'll open another issue about this, but I am fairly certain that the timestamp can be removed.  Its used to order files (but the order seems arbitrary) and when sorted wals are used for recovery a merged sorted view of the files is [constructed](https://github.com/apache/accumulo/blob/a4cc5f82c37b4ff1cb2bb44d99e0b7cef1302fc1/server/tserver/src/main/java/org/apache/accumulo/tserver/log/RecoveryLogsIterator.java#L113) in which order does not matter.  I think the order of the files used to matter, but no longer does and this is vestigial code.

Reply:
Opened #3922